server/container/guice/cassandra-rabbitmq-guice/src/main/java/org/apache/james/modules/TaskSerializationModule.java

```diff
@@ -331,4 +344,11 @@
     public AdditionalInformationDTOModule<?, ?> userReindexingAdditionalInformation(MailboxId.Factory mailboxIdFactory) {
         return UserReindexingTaskAdditionalInformationDTO.serializationModule(mailboxIdFactory);
     }
+
+    @Named("EventNestedTypes")
+    @Provides
+    public Set<DTOModule<?, ?>> eventNestedTypes(Set<AdditionalInformationDTOModule<?, ?>> additionalInformationDTOModules,
+                                            Set<TaskDTOModule<?, ?>> taskDTOModules) {
+        return Sets.union(additionalInformationDTOModules, taskDTOModules);
```

Comment:
it's not only nested types, is it?

Reply:
It is nested types in the context of events